### PR TITLE
Fix brace completion in single quote

### PR DESCRIPTION
**Pull request by abhishekgahlot**
I think this should be this ,
Related issue :https://github.com/ajaxorg/ace/issues/3030
Provide the feedback required or if this change is bad.

### CODE REVIEW

File: lib/ace/mode/c_cpp_highlight_rules.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ var c_cppHighlightRules = function() {
                 next : "comment"
             }, {
                 token : "string", // character
-                regex : "'(?:" + escapeRe + "|.)'"
+                regex : "['](?:" + escapeRe + "|.)*?[']"
```
Comment (nightwing) [inline lib/ace/mode/c_cpp_highlight_rules.js]: In c strings with single quote can have only one character/escape sequense, perhaps this should be 
```js
 "'(?:" + escapeRe + "|.)?'"
```
 instead?

Comment (abhishekgahlot) [inline lib/ace/mode/c_cpp_highlight_rules.js]: Thanks missed that :)

**Comment by nightwing** [comment]
LGTM.
Could you please fill out cla form linked from https://github.com/ajaxorg/ace/blob/master/CONTRIBUTING.md?

**Comment by abhishekgahlot** [comment]
@nightwing Done. Is it recently added I didn't filled any form in my last pull request which was sometime in Jan.

**Comment by nightwing** [comment]
Thanks!
CLA is not new, we just don't need it for onetime small fixes.

**Pull request merged**